.github/workflows/development.yml

```diff
@@ -1,6 +1,8 @@
 name: Development
 
 on: [push, pull_request]
+env:
+  ruby-cache-version: 'v1'
```

Comment:
Looking good, do you want to introduce `with: cache-version: 'v1'`? Is that the plan, is it required?

I do similar test setups and don't seem to have a problem, since we use different gemfiles.

Reply:
Ohh, that is just leftover, I'll remove it for now.